sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2642,9 +2642,6 @@ class HiveDDLSuite
 
       sql("CREATE TABLE t3 (v VOID) USING hive")
       checkAnswer(sql("SELECT * FROM t3"), Seq.empty)
-
-      sql("CREATE TABLE t4 (v VOID)")
-      checkAnswer(sql("SELECT * FROM t4"), Seq.empty)
```

Comment:
This is a duplication in terms of Hive table capability test.
Instead, this is a test coverage of `spark.sql.legacy.createHiveTableByDefault` which is covered already in SQL module like `PlanResolutionSuite`. We can remove this safely.